scripts/python/migrate_data_2024.py

```diff
@@ -0,0 +1,2683 @@
+                    if row[1] not in duplicated_pmids and row[2] is not None and row[2] != '':
+                        # Check if publication is being used
+                        cursor.execute(sql_query_gfd, [row[1]])
+                        data_gfd_2 = cursor.fetchall()
+                        if len(data_gfd_2) != 0:
+                            publications_data[row[0]] = { 'pmid':row[1],
+                                           'title':row[2],
+                                           'source':row[3] }
+
+    except Error as e:
+        print("Error while connecting to MySQL", e)
+    finally:
+        if connection.is_connected():
+            cursor.close()
+            connection.close()
+
+    return publications_data
+
+"""
+    Returns phenotypes
+"""
+def dump_phenotype(host, port, db, user, password):
+    result = {}
+
+    sql_query_user = f""" SELECT phenotype_id, stable_id, name, description, source
+                          FROM phenotype """
+    
+    connection = mysql.connector.connect(host=host,
+                                         database=db,
+                                         user=user,
+                                         port=port,
+                                         password=password)
+
+    try:
+        if connection.is_connected():
+            cursor = connection.cursor()
+            cursor.execute(sql_query_user)
+            data = cursor.fetchall()
+            if len(data) != 0:
+                for row in data:
+                    result[row[0]] = { 'stable_id':row[1],
+                                       'name':row[2],
+                                       'description':row[3],
+                                       'source':row[4] }
+
+    except Error as e:
+        print("Error while connecting to MySQL", e)
+    finally:
+        if connection.is_connected():
+            cursor.close()
+            connection.close()
+
+    return result
+
+"""
+    Return organ data
+    This data is going to be stored as historical data
+"""
+def dump_organ(host, port, db, user, password):
+    result = {}
+
+    sql_query_user = f""" SELECT organ_id, name
+                          FROM organ """
+    
+    connection = mysql.connector.connect(host=host,
+                                         database=db,
+                                         user=user,
+                                         port=port,
+                                         password=password)
+
+    try:
+        if connection.is_connected():
+            cursor = connection.cursor()
+            cursor.execute(sql_query_user)
+            data = cursor.fetchall()
+            if len(data) != 0:
+                for row in data:
+                    result[row[0]] = row[1]
+
+    except Error as e:
+        print("Error while connecting to MySQL", e)
+    finally:
+        if connection.is_connected():
+            cursor.close()
+            connection.close()
+
+    return result
```

Comment:
Should be organs_data

Reply:
This variable is only used inside this function, a generic name `result` will only create confusion outside.